decidim-core/app/views/decidim/participatory_process_steps/_step.html.erb

```diff
@@ -0,0 +1,21 @@
+<% # TODO: Move to decorator? %>
```

Comment:
We don't have decorators anywhere right now, so helpers are the way to go. We're kind of scoping by gem, so decorators feel unnecessary. But I'm in favor of discussing the use case for decorators in the future.